src/clusterfuzz/stacktraces/constants.py

```diff
@@ -185,7 +185,11 @@
     r'.*Sanitizer CHECK failed[:]\s*[^ ]*\s*(.*)')
 SAN_CRASH_TYPE_ADDRESS_REGEX = re.compile(
     r'[ ]*([^ ]*|Atomic [^ ]*) of size ([^ ]*) at ([^ ]*)')
-SAN_DEADLYSIGNAL_REGEX = re.compile(r'.*:DEADLYSIGNAL')
+SAN_DEADLYSIGNAL_REGEX = re.compile(
+    r'(Address|Leak|Memory|UndefinedBehavior|Thread)Sanitizer:DEADLYSIGNAL')
+CONCATENATED_SAN_DEADLYSIGNAL_REGEX = re.compile(
+    r'\n(.+)(' + SAN_DEADLYSIGNAL_REGEX.pattern + r')\n')
```

Comment:
To make this more robust, can we match on any non-space characters? i.e. `[^\s]` instad of `.`.

Reply:
Good point, thanks!
I am using `[^\n]*\S[^\n]*` now to ensure there is a least one non-space char and none of the rest is a newline char.
It is a few seconds slower on testcase `test_syzkaller_kasan` (which has a large test data file) but I hope that is acceptable?